thread_pool.c

```diff
@@ -39,11 +39,12 @@ thread_pool_join(void) {
 
 int
 main(void) {
-    queue_init(&queue);
+    queue = queue_create();
```

Comment:
Check for a returned `NULL` value here, to check if the queue could properly be created.

Reply:
Done.